setup.sh

```diff
@@ -69,10 +67,10 @@ make install
 echo 'PATH=$PATH:/usr/local/pfring/bin:/usr/local/pfring/sbin' >> /etc/bash.bashrc
 
 #Get BRO
-cd ../../
+cd ../../../
```

Comment:
Bro was ending up within the PF_RING source code directory. This brings it to the same level (inside of bro-install).

Reply:
Good catch. d'oh